config/locales/en/casework.yml

```diff
@@ -168,7 +168,24 @@ en:
     will_benefit_from_trust_fund: Does your client stand to benefit from a trust fund inside or outside the UK?
     trust_fund_amount_held: Enter the amount held in the fund
     trust_fund_yearly_dividend: Enter the yearly dividend
-
+    properties_title: Assets
+    properties:
+      property_type: Which assets does your client own or part-own inside or outside the UK?
+      house_type: Which type of property is it?
+      bedrooms: How many bedrooms are there?
+      size_in_acres: What size is the land in acres?
+      usage: How is the %{asset} used?
+      value: How much is the %{asset} worth?
+      outstanding_mortgage: How much is left to pay on the mortgage?
+      percentage_applicant_owned: What percentage of the %{asset} does your client own?
+      percentage_partner_owned: What percentage of the %{asset} does your client’s partner own?
+      is_home_address: Is the address of the %{asset} the same as your client’s home address?
+      has_other_owners: Does anyone else own part of the %{asset}?
+      address: What is the address of your client’s %{asset}?
+    property_owners:
+      name: What is the name of the %{index} other owner?
+      relationship: What is their relationship to your client?
+      percentage_owned: What percentage of the %{asset} do they own?
```

Comment:
The review designs have different headings for the rows to the apply screens: https://www.figma.com/file/thkvnDBbQbHlSqzt1TE3lk/Crime-Apply?type=design&node-id=8359-48221&mode=design&t=k8CcQlVoOSw0ll4P-0